pkg/kyma/cmd/provision/minikube.go

```diff
@@ -233,27 +233,23 @@ func initializeMinikubeConfig() error {
 }
 
 func startMinikube(o *MinikubeOptions) error {
-	virtualSwitchArg := ""
-	if o.VMDriver == vmDriverHyperv {
-		virtualSwitchArg = "--hyperv-virtual-switch=" + o.HypervVirtualSwitch
-	}
-
 	startCmd := []string{"start",
 		"--memory", o.Memory,
 		"--cpus", o.CPU,
-		"--extra-config=apiserver.Authorization.Mode=RBAC",
-		"--extra-config=apiserver.GenericServerRunOptions.CorsAllowedOriginList='.*'",
-		"--extra-config=controller-manager.ClusterSigningCertFile='/var/lib/localkube/certs/ca.crt'",
-		"--extra-config=controller-manager.ClusterSigningKeyFile='/var/lib/localkube/certs/ca.key'",
-		"--extra-config=apiserver.admission-control='LimitRanger,ServiceAccount,DefaultStorageClass,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota'",
+		"--extra-config=apiserver.authorization-mode=RBAC",
+		"--extra-config=apiserver.cors-allowed-origins='http://*'",
```

Comment:
Will it work for https?

Reply:
Shouldn't be a problem, apiserver is not hidden by SSL anyway. That's how Kyma is being installed right now.